### ISSUE TITLE
MRTK3 and Vuforia bug

### ISSUE THREAD

**Issue by Faith012** [PTCInc/vuforia-engine#108]
**Summarize the bug:**
Hololens 2 application with MRTK3 and Vuforia Image Target is not working properly. Gameobject from the Image Target is going somewhere very far away.

**Observed problem**
I tried to create an Image Target for an MRTK3 project and when in Play Mode or deployed on Hololens 2 device my Image Target is detected good but the Gameobject which has to be above the image target is displayed normally for a small ammount of time and then is going somewhere very far and when I try in Play Mode to change its position to see it I can find it but it's scaled as a very small gameobject.

**Expected result**
As a result I want to see my child Gameobject from the Image Target parent in the location I set to bet and to be scaled properly.

**Reproducible steps**
1. Go to Unity and switch to UWP app and then import all MRTK featuer with Microsoft MixedReality Feature Tool
2. Configure the MRTK project as recommended
3. Add Vuforia Engine and configure as recommended
4. Import Image Target database
5. Add MRTK XR Rig and MRTK Input Simulator
6. Configure main camera and camera offset from MRTK XR Rig as recommended
7. Add a Vuforia Image Target and append a child Gameobject which has to be displayed when image is detected (In my case is a primitive Cube)
8. Run application in Play Mode and try to detect the Image

**Affected Vuforia Engine version:**
v10.22.5

**Affected platform:**
 - Workstation OS: Windows
 - Platform Unity
 - Platform version 2022.3.22.f1

**Affected device:**
 - Name: Hololens 2

**Screenshots / Video Recordings**
My scene hierararchy:
<img width="1280" alt="image" src="https://github.com/PTCInc/vuforia-engine/assets/37242337/4acc9a71-3658-4550-93eb-0ad34b588c0f">

**Additional comments**
I found from where the problem is coming. In the main camera of the MRTK XR Rig is a component named "Tracked Pose Driver" which has a component called "Update Type" with the default value of Update and Before Render. If the value is changed to just Update then app is working well. My question about this solution is which problems can create this change and what cost is payed for that.

Parameter which has to be changed:
<img width="1280" alt="Screenshot 2024-03-26 112517" src="https://github.com/PTCInc/vuforia-engine/assets/37242337/05f36206-21d5-4fc6-98f9-b2274b6a18e1">

**Comment by ptc-epassaro** [comment]
Hi @Faith012, great to know that it works! One more tip: if you disable just the Camera simulation in the `MRTKInputSimulator` you should be able to have Vuforia tracking and input testing at the same time. It's still a bit odd and it takes some getting used to, but it could be something useful for the future, in case you need it.
![Screenshot 2024-03-27 144749](https://github.com/PTCInc/vuforia-engine/assets/58174995/59bad7b8-d203-4064-86a4-8da8afd18128)

Regarding the jitter, does this happen in PlayMode or on HoloLens 2?
In general, jitter can have multiple causes. The most common is that the scale of the physical target does not match with the image in the database, but there can be more factors. These pages in the Vuforia documentation can be useful:
- [Best Practices for Designing and Developing Image-Based Targets](https://developer.vuforia.com/library/objects/best-practices-designing-and-developing-image-based-targets)
- [Image Targets Optimization Techniques](https://developer.vuforia.com/library/objects/image-targets-optimization-techniques)
- [The Physical Properties of Image-Based Targets](https://developer.vuforia.com/library/objects/physical-properties-image-based-targets)
- [Enable Device Tracking in Unity](https://developer.vuforia.com/library/device-tracking/enable-device-tracking-unity)

One more thing that can be helpful is to enable pose smoothing on the `DefaultObserverEventHandler` component:
![Screenshot 2024-03-27 151833](https://github.com/PTCInc/vuforia-engine/assets/58174995/fb4f641e-d4d2-4c7d-9482-f076253720d6)

**Comment by Faith012** [comment]
Regarding the tip, if I disable Camera simulation then I can simulate the hands and whaterver functionalities I need but only before the target is tracked. After that. simulated hands for example dissapears and my gamobject too, it feels like MRTK Input Simulator wants to overlay inputs over the Vuforia and cannot do that but anyway, the right solution for me remains to deactivate MRTK Input Simulator while I check the functionalities of Vuforia. 
As for jitter, I saw that I had a typo when I set up the width and after resolving this and added smooth transition, jitter is nearly visible.
Thanks a lot for help! I thinked I would never resolve this bug :)

Should I close this issue now?
